ui/src/layouts/storage.rs

```diff
@@ -831,7 +832,20 @@ async fn drag_and_drop_function(
                 window.eval(&script);
             }
             FileDropEvent::Dropped(files_local_path) => {
+                #[cfg(not(target_os = "linux"))]
                 ch.send(ChanCmd::UploadFiles(files_local_path));
+                #[cfg(target_os = "linux")]
```

Comment:
Instead of a double feature gate here, what could be done is is to use the `cfg!` macro inside `decoded_path_string` to either replace or passthrough the content on other platforms.